sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/login/AADLoginIT.java

```diff
@@ -0,0 +1,128 @@
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserService;
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.function.Consumer;
+import java.util.regex.Pattern;
+
+import static com.azure.test.oauth.OAuthUtils.AAD_MULTI_TENANT_CLIENT_ID;
+import static com.azure.test.oauth.OAuthUtils.AAD_MULTI_TENANT_CLIENT_SECRET;
+import static com.azure.test.oauth.OAuthUtils.AAD_TENANT_ID_1;
+import static com.azure.test.oauth.OAuthUtils.AAD_USER_NAME_1;
+import static com.azure.test.oauth.OAuthUtils.AAD_USER_PASSWORD_1;
+import static org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated;
+
+public class AADLoginIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AADLoginIT.class);
+
+    static {
+        final String chromedriverLinux = "src/test/resources/driver/chromedriver_linux64";
+        final String chromedriverWin32 = "src/test/resources/driver/chromedriver_win32.exe";
+        final String chromedriverMac = "src/test/resources/driver/chromedriver_mac64";
+        String osName = System.getProperty("os.name").toLowerCase();
+        if (Pattern.matches("linux.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverLinux);
+        } else if (Pattern.matches("windows.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverWin32);
+        } else if (Pattern.matches("mac.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverMac);
+        }
+    }
+
+//    @Test
+    public void loginTest() {
+        this.runTest(app -> {
+            WebDriver driver = new ChromeDriver();
+            WebDriverWait wait = new WebDriverWait(driver, 10);
+            driver.get(app.root() + "api/group1");
+            wait.until(presenceOfElementLocated(By.name("loginfmt")))
+                .sendKeys(System.getenv(AAD_USER_NAME_1) + Keys.ENTER);
+            wait.until(presenceOfElementLocated(By.name("passwd")))
+                .sendKeys(System.getenv(AAD_USER_PASSWORD_1));
+
+            try {
+                Thread.sleep(3000);
+                driver.findElement(By.cssSelector("input[type='submit']")).click();
+                Thread.sleep(3000);
+                driver.findElement(By.cssSelector("input[type='submit']")).click();
+                Thread.sleep(3000);
+                Assert.assertEquals("group1", driver.findElement(By.tagName("body")).getText());
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+            } finally {
+                driver.quit();
+            }
+        });
+    }
+
+    public void runTest(Consumer<AppRunner> command) {
+        try (AppRunner app = new AppRunner(AADLoginIT.DumbApp.class)) {
```

Comment:
Add `// TODO` here, just remind us to delete old configuration item after new aad implementation start to work.